frontend/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,7 @@
+import Home from '../components/home/Home.tsx';
+
+function HomePage(): JSX.Element {
+  return <Home />;
```

Comment:
You only need and make use of 1 component, so no need to nest as a separate component in `components`